### DISCUSSION TITLE
How to remove the default styles applied to a storybook page?

### DISCUSSION THREAD

**Discussion by tishoyanchev** [storybookjs/storybook#26866] [Help] [answered]
### Summary

I have created a new page in my Storybook which loads an mdx file. 

<img width="217" alt="one" src="https://github.com/storybookjs/storybook/assets/108734092/5bf896ec-e42a-4ea6-9f7d-ddb2b2fa6064">

In it, I am referencing a web component. 

My mdx file.
```
import { Meta, Description } from '@storybook/blocks';
import {Story, Source, Controls, Canvas,} from '@storybook/blocks';
import dedent from 'ts-dedent';

<Meta title="DDS Components Overview" parameters={{ previewTabs: { canvas: { hidden: true }, 'storybook/docs/panel': { hidden: true }, 'usage-tab/panel': { hidden: true } }, layout: 'fullscreen' }} />

<ifx-overview-table />
```

The problem is that there are some default styles applied to storybook which add padding and other formatting related values, which is generally good, but for this page, I need to get rid of them. 

Using the inspect tool, I can see which styles are applied, but I cannot see where do they exist in the project. 

<img width="739" alt="three" src="https://github.com/storybookjs/storybook/assets/108734092/02d42360-f835-4afd-a144-1d3d0e42cb85">

I want to remove these styles, especially the `padding`, and then add `overflow: hidden` on that wrapper in order to hide the 
page scrollbars, because my table has its own scrollbars. 
However, as mentioned, I only want to do this for this one page, not for all of my other pages. 

Is there a way to do this?

I only tried adding `layout: 'fullscreen'` on my `<meta>` tag, in my `stories` file, and in the `preview.js` but it did not change anything.

**Comment by tishoyanchev** [comment, marked as answer]
I found a workaround. I was able to grab the storybook wrapper by querying the `parentElement` of my component.

**Comment by just-be-weird** [comment]
I was running into a problem where the theme for my custom styled components would get overridden in seemingly random places. This often happened when I was trying to use .mdx files for my Storybook stories. I managed to fix it by using Doc-Block-Unstyled as described in the Storybook documentation. Reference https://storybook.js.org/docs/api/doc-blocks/doc-block-unstyled

```jsx
import { Meta, Unstyled } from '@storybook/blocks';
import { SimplePagination } from './stories/simple-pagination';

<Meta
  title="Packages/Data Grid/ 1. Simple Pagination"
  parameters={{
    options: {
      storySort: {
        order: ['Instructions'],
      },
    },
  }}
/>

<Unstyled>
    <SimplePagination />
</Unstyled>
```
